TEAMS.yaml

```diff
@@ -79,7 +79,7 @@ cockroachdb/cluster-ui:
 cockroachdb/obs-inf-prs:
   triage_column_id: 14196277
 cockroachdb/multi-tenant:
-  triage_column_id: 16575985
+  label: T-multitenant
```

Comment:
does this have the intended effect? i would expect that you'd need to also update the `Team` struct in pkg/internal/team/team.go

Reply:
oh i see it now. i needed a rebase